### ISSUE TITLE
ucar.nc2.NCdumpW generates invalid NcML for HDF4 Vdata

### ISSUE THREAD

**Issue by hyoklee** [Unidata/thredds#148]
Test file: ftp://ftp.hdfgroup.uiuc.edu/pub/outgoing/NASAHDF/1B21.071022.56609.6.HDF

 Test:
    %java -Xmx1g -classpath netcdfAll-4.6.jar ucar.nc2.NCdumpW  1B21.071022.56609.6.HDF -ncml > 1B21.071022.56609.6.HDF.ncml

 Output:
```
  <group name='DATA_GRANULE' >
    <structure name='PR_CAL_COEF>
      <variable name='transCoef' type='float' />
      <variable name='receptCoef' type='float' />
      <variable name='fcifIOchar' type='float' shape='16' />
    </structure>
```
There's no closing quote(') after 'PR_CAL_COEF.

**Comment by hyoklee** [comment]
Is "structure" a valid NcML tag? I think it should be 
```
  <variable name='PR_CAL_COEF' type='Structure'>
```
like THREDDS/toolsUI does.

I don't know why CLI NcdumpW uses 'structure' tag.

**Comment by cwardgar** [comment]
@hyoklee Agreed, NcdumpW has its own code for write NcML, but it should be the same thing that ToolsUI and others use. I'll get to work on this.

**Comment by hyoklee** [comment]
I found another difference.

THREDDS handles unsigned type attribute correctly:
```
          <variable name="GroundPixelQualityFlags" shape="nCandidate YDim XDim" type="short">
            <attribute name="_FillValue" type="short" value="-1" isUnsigned="true" />
```

netCDF-All doesn't:
```
          <variable name='GroundPixelQualityFlags' type='short' shape='nCandidate YDim XDim' >
            <attribute name='_FillValue' value='-1 ' type='short' />

```

Test file is here:

ftp://ftp.hdfgroup.uiuc.edu/pub/outgoing/NASAHDF/OMI-Aura_L2G-OMCLDO2G_2007m0129_v002-2007m0130t174603.he5

**Comment by cwardgar** [comment]
Fixed by be2c6165a9614923b6cf9fc3cab8db328a4d7b47

Note that those changes will appear in v5.0.0, not v4.6.
